### DISCUSSION TITLE
Hi-res fix, am I using it right?

### DISCUSSION THREAD

**Discussion by fiveseven808** [Kilvoctu/aiyabot#152] [Q&A]
I can specify the upscaler used by the hires fix, but I can't specify any of the other options like hires steps, denoising strength or the upscaling multiplier. 

Are these options hidden somewhere? will they be available in the future?

**Comment by Kilvoctu** [comment]
AIYA uses only the legacy hires fix as mentioned on readme and somewhere in my wiki, so she's set up to emulate how it was done in the Web UI. As such, the multiplier will always be 2x and hires steps is half of steps.  
Denoising strength, you can set that with **strength**, I believe (it applies to both hires fix and init_img).  
I could add a way to set the steps, I suppose 🤔; it would be like how user [sets multiplier for extra networks](https://github.com/Kilvoctu/aiyabot/wiki/draw-command#extra-net). Shouldn't be too hard.
